src/routes/+page.svelte

```diff
@@ -33,27 +33,30 @@
 	const currentDate = new Date();
 </script>
 
-<!-- Top bar with logo and "Get early access" button  -->
-<section
-	class={`bg-gradient-to-r from-neutral-800 from-25% via-neutral-400 to-neutral-800 to-75% pb-0.5 text-neutral-100 h-[${navSize}px]`}>
-	<div class="flex items-center bg-hero p-4 h-full">
-    	<div class="h-4 md:h-6 grow"><Logo text={true} /></div>
-    	<a href="https://tally.so/r/nrVBgN"><button class="rounded-full border-2 border-neutral-700 px-4 py-1">Get Early Access</button></a>
-	</div>
-</section>
-
-<!-- Hero -->
-<section id="hero" class={`py-8 bg-hero bg-[size:200%] md:bg-cover min-h-[calc(100vh-${navSize}px)]`}>
-	<div class="flex flex-col items-center gap-4 px-[10%]">
-		<Text type="callout" color="darkPrimary" class="rounded-full px-4 sm:px-5 py-2 bg-primary-dark">Backed by top investors</Text>
-		<Text type="hero" color="darkPrimary" class="sm:max-w-[60%] text-center">Turn all Partnerships into Revenue</Text>
-		<Text type="subhead" color="darkSecondary" class="sm:max-w-[55%] text-center">
-			Our AI Partnerships Intelligence Platform analyzes sales calls to uncover partner
-			opportunities. Our insights and workflows empower you to boost revenue and close deals faster
-			through partnerships.
-		</Text>
-		<a href="https://tally.so/r/nrVBgN"><button class="rounded-full bg-orange-500 px-4 py-1 text-neutral-100">Get Early Access</button></a>
-	</div>
+<!-- Above the fold section -->
+<section class="bg-hero">
+	<!-- Top bar with logo and "Get early access" button  -->
+	<section class="mx-4 sm:mx-28 flex flex-col flex-nowrap justify-between items-stretch">
+		<div class="flex flex-row flex-nowrap justify-between items-center py-4 sm:py-9">
```

Comment:
I believe we should be using `py-9 md:py-4` instead because of the way TailwindCSS defines breakpoints . Looking at [the documentation](https://tailwindcss.com/docs/screens) on these, if we use `py-4 sm:py-9` I believe the `py-9` class will actually apply to all screen sizes, we need to instead design for mobile-first and apply the desktop styles as the "exception".

Not sure if this was intended but I believe this might be why the navbar looks big on desktop with this Pull Request as is 🔽 

<img width="1782" alt="SCR-20240811-ckpy" src="https://github.com/user-attachments/assets/62445917-2901-4c85-a291-97c7b98e70ac">

In mobile looks nice though ✅ 

<img width="390" alt="SCR-20240811-ckox" src="https://github.com/user-attachments/assets/3ca31e1f-7ba1-4df4-86ef-db0b3ab1a9ab">